prometheus-pusher/main_test.go

```diff
@@ -46,3 +55,56 @@ func TestMergeLabel(t *testing.T) {
 	assert.Len(t, pl.Labels, 3)
 
 }
+
+func TestPusher(t *testing.T) {
+	// Fake a Pushgateway that responds with 202 to DELETE and with 200 in
+	// all other cases.
```

Comment:
This test case just make sure Prometheus push is pushing the metrics in right format